exporter/importables.go

```diff
@@ -1324,6 +1324,33 @@ var resourcesMap map[string]importable = map[string]importable{
 			{Path: "evaluation.notification.subscriptions.user_email", Resource: "databricks_user", Match: "user_name", MatchType: MatchCaseInsensitive},
 		},
 	},
+	"databricks_apps_settings_custom_template": {
+		WorkspaceLevel:  true,
+		PluginFramework: true,
+		Service:         "apps",
+		Name:            func(ic *importContext, d *schema.ResourceData) string { return d.Id() },
+		List:            listAppsSettingsCustomTemplates,
+		Ignore:          generateIgnoreObjectWithEmptyAttributeValue("databricks_apps_settings_custom_template", "name"),
+	},
+	"databricks_app": {
+		WorkspaceLevel:  true,
+		PluginFramework: true,
+		Service:         "apps",
+		Name:            func(ic *importContext, d *schema.ResourceData) string { return d.Id() },
+		List:            listApps,
+		Import:          importApp,
+		Ignore:          generateIgnoreObjectWithEmptyAttributeValue("databricks_app", "name"),
+		Depends: []reference{
+			{Path: "resources.sql_warehouse.id", Resource: "databricks_sql_endpoint"},
+			{Path: "resources.serving_endpoint.name", Resource: "databricks_model_serving"},
+			{Path: "resources.job.id", Resource: "databricks_job"},
+			{Path: "resources.secret.scope", Resource: "databricks_secret_scope"},
+			{Path: "resources.secret.key", Resource: "databricks_secret", Match: "key",
+				IsValidApproximation: createIsMatchingScopeAndKey("scope", "key")},
+			{Path: "resources.uc_securable.securable_full_name", Resource: "databricks_volume"},
+			// {Path: "budget_policy_id", Resource: "databricks_budget"},
```

Comment:
Is this unintentionally left here?

Reply:
yes, it's not implemented yet - just a placeholder for future implementation